packages/notifi-react-card/lib/components/NotifiEmailInput.tsx

```diff
@@ -92,7 +94,7 @@ export const NotifiEmailInput: React.FC<NotifiEmailInputProps> = ({
           classNames?.container,
         )}
       >
-        <EmailIcon className={'NotifiInput__icon'} />
+        {hideStartIcon ? null : <EmailIcon className={'NotifiInput__icon'} />}
```

Comment:
now that i think about it, i think this should be done by overriding CSS --

the user can provide a stylesheet with
```
.NotifiInput__icon {
  display: none;
}
```

I think this is more flexible than a prop, since it's harder to provide a specific prop to the inputs if you're using the Subscription card. wdyt?